angular/test/protractor/helpers/page_helper.coffee

```diff
@@ -13,7 +13,6 @@ module.exports = new class PageHelper
   loadPath: (path) ->
     browser.get('development/'+path)
     browser.driver.manage().window().setSize(1280, 1024)
-    browser.executeScript("document.querySelector('.lmo-navbar').style.position = 'absolute'")
```

Comment:
I thought this was preventing us from getting weird 'you clicked on the navbar' errors in test?